### ISSUE TITLE
Fix Docs

### ISSUE THREAD

**Issue by scetron** [nautobot/nautobot-app-device-onboarding#107]
### Environment
* Python version:  3.8
* Nautobot version:  x
* nautobot-device-onboarding version: 1.6 / develo

### Steps to Reproduce
1. `inv build debug` from a fresh clone
2. Docs container fails due to missing modules
3.

### Expected Behavior
Docs container builds successfully

### Observed Behavior

Docs container fails with this traceback due to missing dependencies.

```
nautobot-device-onboarding-docs-1      | ERROR    -  Error reading page 'dev/code_reference/api.md': No module named 'mkdocstrings.handlers.python'
nautobot-device-onboarding-docs-1      | Traceback (most recent call last):
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocstrings/handlers/base.py", line 563, in get_handler
nautobot-device-onboarding-docs-1      |     module = importlib.import_module(f"mkdocstrings_handlers.{name}")
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
nautobot-device-onboarding-docs-1      |     return _bootstrap._gcd_import(name[level:], package, level)
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocstrings_handlers/python/__init__.py", line 3, in <module>
nautobot-device-onboarding-docs-1      |     from mkdocstrings_handlers.python.handler import get_handler
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocstrings_handlers/python/handler.py", line 13, in <module>
nautobot-device-onboarding-docs-1      |     from griffe.agents.extensions import load_extensions
nautobot-device-onboarding-docs-1      | ModuleNotFoundError: No module named 'griffe.agents.extensions'
nautobot-device-onboarding-docs-1      | 
nautobot-device-onboarding-docs-1      | During handling of the above exception, another exception occurred:
nautobot-device-onboarding-docs-1      | 
nautobot-device-onboarding-docs-1      | Traceback (most recent call last):
nautobot-device-onboarding-docs-1      |   File "/usr/local/bin/mkdocs", line 8, in <module>
nautobot-device-onboarding-docs-1      |     sys.exit(cli())
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1157, in __call__
nautobot-device-onboarding-docs-1      |     return self.main(*args, **kwargs)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1078, in main
nautobot-device-onboarding-docs-1      |     rv = self.invoke(ctx)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1688, in invoke
nautobot-device-onboarding-docs-1      |     return _process_result(sub_ctx.command.invoke(sub_ctx))
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1434, in invoke
nautobot-device-onboarding-docs-1      |     return ctx.invoke(self.callback, **ctx.params)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 783, in invoke
nautobot-device-onboarding-docs-1      |     return __callback(*args, **kwargs)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocs/__main__.py", line 181, in serve_command
nautobot-device-onboarding-docs-1      |     serve.serve(dev_addr=dev_addr, livereload=livereload, watch=watch, **kwargs)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocs/commands/serve.py", line 63, in serve
nautobot-device-onboarding-docs-1      |     config = builder()
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocs/commands/serve.py", line 58, in builder
nautobot-device-onboarding-docs-1      |     build(config, live_server=live_server, dirty=dirty)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocs/commands/build.py", line 292, in build
nautobot-device-onboarding-docs-1      |     _populate_page(file.page, config, files, dirty)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocs/commands/build.py", line 174, in _populate_page
nautobot-device-onboarding-docs-1      |     page.render(config, files)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocs/structure/pages.py", line 175, in render
nautobot-device-onboarding-docs-1      |     self.content = md.convert(self.markdown)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/markdown/core.py", line 264, in convert
nautobot-device-onboarding-docs-1      |     root = self.parser.parseDocument(self.lines).getroot()
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/markdown/blockparser.py", line 90, in parseDocument
nautobot-device-onboarding-docs-1      |     self.parseChunk(self.root, '\n'.join(lines))
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/markdown/blockparser.py", line 105, in parseChunk
nautobot-device-onboarding-docs-1      |     self.parseBlocks(parent, text.split('\n\n'))
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/markdown/blockparser.py", line 123, in parseBlocks
nautobot-device-onboarding-docs-1      |     if processor.run(parent, blocks) is not False:
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocstrings/extension.py", line 121, in run
nautobot-device-onboarding-docs-1      |     html, handler, data = self._process_block(identifier, block, heading_level)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocstrings/extension.py", line 175, in _process_block
nautobot-device-onboarding-docs-1      |     handler = self._handlers.get_handler(handler_name, handler_config)
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/site-packages/mkdocstrings/handlers/base.py", line 565, in get_handler
nautobot-device-onboarding-docs-1      |     module = importlib.import_module(f"mkdocstrings.handlers.{name}")
nautobot-device-onboarding-docs-1      |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
nautobot-device-onboarding-docs-1      |     return _bootstrap._gcd_import(name[level:], package, level)
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
nautobot-device-onboarding-docs-1      |   File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
nautobot-device-onboarding-docs-1      | ModuleNotFoundError: No module named 'mkdocstrings.handlers.python' 
```

**Comment by pawamoy** [comment]
Looks it simply need dependency upgrades of mkdocstrings-python and/or Griffe.

**Comment by scetron** [comment]
Belied to be fixed, at least in #214
